Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Document.java

```diff
@@ -186,16 +186,20 @@ public void setServiceStopTime(Date serviceStopTime) {
     /**
      * @return the size
      */
-    public double getSize() {
-        return Double.parseDouble(new DecimalFormat("##.##").format(size / 1024));
+    public int getSize() {
+        return size / 1024;
```

Comment:
Will the size always be 1024 divisible, or is the "div" value acceptable?  Please reflect the answer to this in the Javadoc for this function.